sdcm/cluster.py

```diff
@@ -2333,7 +2333,7 @@ def _resharding_status(self, status):
         Check is there's Reshard listed in the log
```

Comment:
Need to chaget doc string, that searching in nodetool output

Reply:
PR  is not in status "ready for review" because of @roydahan wanted me to run 1TB longevity. So longevity yaml is not part of PR and will be reverted after I'll finish the test. So, this comment is not relevant

Reply:
will do, thanks

Reply:
Done